doc/classes/TileMapLayer.xml

```diff
@@ -22,6 +22,15 @@
 				[b]Note:[/b] If the properties of [param tile_data] object should change over time, use [method notify_runtime_tile_data_update] to notify the [TileMapLayer] it needs an update.
 			</description>
 		</method>
+		<method name="_update_cells" qualifiers="virtual">
+			<return type="void" />
+			<param index="0" name="coords" type="Vector2i[]" />
+			<param index="1" name="forced_cleanup" type="bool" />
+			<description>
+				Called when this [TileMapLayer]'s cells need an internal update. This update may be caused from individual cells being modified or by a change in the [member tile_set] (causing all cells to be queued for an update). The first call to this function is always for initializing all the [TileMapLayer]'s cells. [param coords] contains the coordinates of all modified cells, roughly in the order they were modified. [param forced_cleanup] is [code]true[/code] when a cleanup is required. This usually means that the [TileMapLayer] has either left the canvas, has left the tree or has been freed. It may also be [code]true[/code] if an update is requested when either [member tile_set] is [code]null[/code], [member enabled] is [code]false[/code] or this [TileMapLayer] is outside of the [SceneTree]. See also [method update_internals].
+				[b]Warning:[/b] Implementing this method may degrade the [TileMapLayer]'s performance.
```

Comment:
What does it mean when "a cleanup is required"? You do explain well when and why it happens, but you don't explain how the user should handle this.

Reply:
A "cleanup" is needed when the TileMapLayer internals should be cleaned up. That mostly means:
- When layer is disabled
- The TileSet is null
- The layer is not visible
- The TileMapLayer quits the tree